charts/policy-reporter/charts/monitoring/values.yaml

```
@@ -7,6 +7,16 @@ serviceMonitor:
   # labels to match the serviceMonitorSelector of the Prometheus Resource
   labels: {}
 
+grafana:
+  dashboards:
+    # Label to find dashboards using the k8s sidecar
+    label: grafana_dashboard
+  folder:
+    # Annotation to enable folder storage using the k8s sidecar
+    annotation: grafana_folder
+    # Grafana folder in which to store the dashboards
+    name: kyverno
```

I know that the primary use case of Policy Reporter is currently the combination with Kyverno but in its core, Policy Reporter Dashboards are useful for all Tools using the PolicyReport CRDs. So I would prefer to use a more generic default.

```suggestion
    name: Policy Reporter
```